public/locales/en.json

```diff
@@ -290,6 +290,7 @@
             "error": {
                 "openIdLogin": "Invalid OpenID provided",
                 "emailLogin": "Invalid email or password",
+                "emailRegister": "Signup via email failed. Maybe there is already an account with that email address.",
```

Comment:
I think failed implies that something went wrong and I should try it again later but if the mail address is already in use this is not the case. If there is no other reason for the registration to fail I think we can say that the address is already in use.

Reply:
I tested the current 1.6 frontend and it also does this distinction. I therefore agree that we need two different error messages and so a bit more complex error handling.